buildAndReleaseTask/tests/envvars_storage.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
```

Comment:
Thank you for writing a test for this! 👍

Reply:
No worries, but the codebase now have two tests that are 90% the same code, so they will need some refactor-love in the future.

Reply:
Yes, absolutely. I plan on doing that (or if you want to ;)) in a separate PR.